doc/releases/changelog-dev.md

```diff
@@ -185,4 +211,4 @@
 
 This release contains contributions from (in alphabetical order):
 
-Olivia Di Matteo, Josh Izaac, Christina Lee, David Wierichs.
\ No newline at end of file
+Olivia Di Matteo, Josh Izaac, Christina Lee, David Wierichs, Utkarsh.
```

Comment:
Is this how you prefer to be referred to in a professional setting? Up to you, but I recommend adding your last name too